modules/ROOT/pages/clustering/clustering-advanced/reconciler.adoc

```diff
@@ -0,0 +1,58 @@
+[role=enterprise-edition]
+[[cluster-reconciler]]
+= Reconciler
+:description: This section describes how changes to the DBMS are processed by each server.
+
+[[cluster-reconciler-introduction]]
```

Comment:
Are all the section labels needed? Do you want specific IDs for each of them? Applies to later sections as well.

Reply:
Ah, I thought I was supposed to label all sections. I don't think deep linking to a section on a page this small is necessarily important, happy to remove them.